specification/provisioningservices/resource-manager/readme.md

```diff
@@ -0,0 +1,57 @@
+# IotHub
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Provisioning Services.
+
+
+
+---
+## Getting Started 
+To build the SDK for ProvisioningServices, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information 
+These are the global settings for the API.
+
+``` yaml
+openapi-type: arm
+tag: package-2017-08
+```
+
+### Tag: package-2017-08
+
+These settings apply only when `--tag=package-2017-08` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-08'
+input-file:
+- Microsoft.Devices/2017-08-21-preview/iothub.json
```

Comment:
bad filename - this should read 
`- Microsoft.Devices/2017-08-21-preview/iotdps.json`